source/documentation/common/backup-restore/proc-Restoring_the_Backup_on_a_New_Self-hosted_Engine.adoc

```diff
@@ -93,7 +93,21 @@ The script creates the virtual machine. This can take some time if the {engine-a
 +
 [NOTE]
 ====
-If the host becomes non operational and the deployment fails (due to missing required network or similar problem), you can repeat the process, and answer _YES_ to the option _Pause the execution after adding this host to the engine?_. Pausing the process allows you to connect to the restored engine and manually review and remediate the configuration before proceeding with the deployment.
+If the host becomes non operational, due to a missing required network or a similar problem, deployment will pause and a message such as the following will be displayed:
```

Comment:
1. non-operational
2. suggestion: rewrite using present simple: "... deployment pauses and a message such as the following is displayed:"

Reply:
non operational (without a hyphen) is what is used in the UI, so I am following that convention in this case.